Documentation/project-docs/standard-platform.md

```diff
@@ -236,6 +236,17 @@ The above `project.json` will cause NuGet to do a compatibiltiy check, enforcing
 }
 ```
 
+## Tool Support
+Tooling support for the `netstandard` TFM is as follows. This table will be updated regularly to reflect current status.
```

Comment:
Open to suggestions, this was intended as a starting point. Not sure what the best term would be here.